### ISSUE TITLE
Error when creating orders

### ISSUE THREAD

**Issue by ArneVerleyenIO** [AcclaroInc/craft-translations#532]
Craft Pro 5.3.4
Plugin version v4.0.0

Error: Setting unknown property: acclaro\translations\records\OrderRecord::programId

I tried creating an order and got the error above. We used the translation service export/import. 
<img width="1463" alt="Screenshot 2024-10-31 at 15 57 56" src="https://github.com/user-attachments/assets/81475e29-d264-4b59-a0d6-3ce173501702">

**Comment by MerrittBoydston** [comment]
Hi @ArneVerleyenIO - to help us investigate further, could you please provide the following information:

1. What specific version of the plugin did you update to?
2. Did you install any beta versions of the plugin or Craft CMS?
3. Can you confirm if the migration process was completed successfully?

Also, could you please give me more details about the steps you took when you encountered this error?

**Comment by ArneVerleyenIO** [comment]
@MerrittBoydston we just installed the new version on a new project.
Craft Pro 5.3.4
Plugin version v4.0.0

**Comment by MicaDelpiano** [comment]
Hi @sharmarishabh1 and @MerrittBoydston,

Are there any updates here? I received a ticket from Jonathan Pardon that says the following:

> Just wanted to give some context with github issue: https://github.com/AcclaroInc/craft-translations/issues/532
>  
> We are trying this import export function as a POC for a new project we are currently working on. The client wants to export json for a translator and import it afterwards again.
>  
> If we cannot confirm this is working by Monday evening, we will have to find another solution.
> 
> Since we are already working with your plugin (for Desotec, a Craft 4 project) we would like to work here as well with the plugin.  Also we will be updating this Desotec Craft 4 project to Craft 5 soon, and hope the issues will be solved by then.
>  
> So Monday is really important deadline for us, or at least we should have some feedback/timing about the issue solution.


Thank you!

**Comment by MerrittBoydston** [comment]
## 🆕 Updates
- Hi @MicaDelpiano - we have a PR for this here: https://github.com/AcclaroInc/craft-translations/pull/534

**Comment by sidedwards** [comment]
@ArneVerleyenIO, can you please update to [`v4.0.1`](https://github.com/AcclaroInc/craft-translations/releases/tag/v4.0.1) and confirm if that resolves your issue?

**Comment by ArneVerleyenIO** [comment]
After installing the update, we got another error when creating an order:
Error: Setting unknown property: acclaro\translations\records\OrderRecord::programId
Checking the database, we the field was indeed non existing.
We decided to remove the plugin and reinstall, and then the field was present and the order was created. But we think this is still a bug for upgrading users.

Extra info:
two tables remained in the database after removing the plugin
In the create order > advanced there's a type saying "Recomended"
